BlahBlahFlat.WebApi/Controllers/PlacementsController.cs

```diff
@@ -110,5 +110,23 @@ public async Task<IActionResult> UpdatePlacementsAsync([FromBody] PlacementDto p
         return NotFound();
     }
 
+    /// <summary>
+    /// Delete placement
+    /// </summary>
+    /// <example>Delete: api/placements/1</example>
+    /// <param name="cancellationToken"><see cref="CancellationToken"/></param>
+    /// <returns>Placement.</returns>
+    /// <response code="204">OK.</response>
+    /// <response code="404">Not found: If placement is missing.</response>
+    [HttpDelete("{id:int}")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [Produces(Application.Json)]
+    public async Task<IActionResult> DeletePlacementAsync([FromBody] int id, CancellationToken cancellationToken)
+    {
+        await _placementService.DeletePlacement(id, cancellationToken);
+        return Ok();
```

Comment:
Should return 204
https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE